integration-tests/src/test/java/oracle/weblogic/kubernetes/ItLivenessProbeCustomization.java

```diff
@@ -0,0 +1,487 @@
+  private static final String CLUSTER_NAME_PREFIX = "cluster-";
+  private static final int replicaCount = 1;
+  private static final int NUMBER_OF_CLUSTERS_MIIDOMAIN = 2;
+  private static final String adminServerPodName = domainUid + "-" + ADMIN_SERVER_NAME_BASE;
+  private static final String managedServerPrefix = domainUid + "-" + MANAGED_SERVER_NAME_BASE;
+  private static final String APPCHECK_SCRIPT = "appcheck.sh";
+  private static final String COPY_CMD = "copy-cmd.txt";
+  private static LoggingFacade logger = null;
+
+  private static ConditionFactory withStandardRetryPolicy =
+      with().pollDelay(2, SECONDS)
+          .and().with().pollInterval(10, SECONDS)
+          .atMost(5, MINUTES).await();
+
+  /**
+   * Get namespaces for operator and WebLogic domain.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+    // get a unique operator namespace
+    logger.info("Getting a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    final String opNamespace = namespaces.get(0);
+
+    // get a unique domain namespace
+    logger.info("Getting a unique namespace for WebLogic domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+
+    // install and verify operator
+    installAndVerifyOperator(opNamespace, domainNamespace);
+
+    // create mii with additional livenessprobecustom script
+    String imageName = createAndVerifyDomainImage();
+
+    // create a basic model in image domain
+    createAndVerifyMiiDomain(imageName);
+  }
+
+  /**
+   * Verify the customization of liveness probe.
+   * Build model in image with liveness probe custom script for a 2 clusters domain.
+   * Enable "LIVENESS_PROBE_CUSTOM_SCRIPT" while creating domain CR.
+   * After domain is created copy the file named tempfile.txt into tested managed server pods in
+   * both clusters, which is used by custom script to trigger liveness probe.
+   * Verify the the container managed server pods in both clusters are restarted
+   */
+  @Test
+  @Order(1)
+  @DisplayName("Test customization of the liveness probe")
+  public void testLivenessProbe() {
+    Domain domain1 = assertDoesNotThrow(() -> getDomainCustomResource(domainUid, domainNamespace),
+        String.format("getDomainCustomResource failed with ApiException when tried to get domain %s in namespace %s",
+            domainUid, domainNamespace));
+    assertNotNull(domain1, "Got null domain resource");
+
+    // create temp file
+    String fileName = "tempFile";
+    String content = "This one line file is to test livenessProbe custom script";
+    File tempFile = assertDoesNotThrow(() -> createTempfile(fileName, content),
```

Comment:
Insert the content directly into the method createTempfile() as a string

Reply:
Done